server/routes/getRewards.ts

```diff
@@ -18,26 +21,20 @@ interface IEventData {
 }
 
 const getRewards = async (models, req: Request, res: Response, next: NextFunction) => {
-  const { chain, stash } = req.query;
+  const { chain, stash_id } = req.query;
```

Comment:
@YameenMalik  please make this ```stash``` instead of ```stash_id```, just to make it consistent with other routes

Reply:
fixed in latest commit